src/SensorVisuals/components/TabCharts/TabCharts.js

```diff
@@ -2,52 +2,44 @@ import React, { Fragment, useEffect, useState } from 'react';
 import { lazy } from 'react';
 import { Grid, Typography } from '@material-ui/core';
 import PropTypes from 'prop-types';
-
 import GatewayChart from '../GatewayChart/GatewayChart';
+
 const NodeVoltage = lazy(() => import('../NodeVoltage/NodeVoltage'));
 const SoilTemp = lazy(() => import('../SoilTemp/SoilTemp'));
-const TempByLbs = lazy(() => import('../LitterbagTemp/LitterbagTemp'));
+const LitterbagTemp = lazy(() => import('../LitterbagTemp/LitterbagTemp'));
 const VolumetricWater = lazy(() => import('../VolumetricWater/VolumetricWater'));
 
 export const convertEpochtoDatetime = (epoch) => {
-  var date = new Date(epoch - new Date().getTimezoneOffset() * 60000);
+  let date = new Date(epoch - new Date().getTimezoneOffset() * 60000);
   // eslint-disable-next-line
-  var iso = date.toISOString().match(/(\d{4}\-\d{2}\-\d{2})T(\d{2}:\d{2}:\d{2})/);
+  let iso = date.toISOString().match(/(\d{4}\-\d{2}\-\d{2})T(\d{2}:\d{2}:\d{2})/);
   return iso[1] + ' ' + iso[2];
 };
 
 const now = convertEpochtoDatetime(Date.now());
 
-const TabCharts = (props) => {
+const TabCharts = ({ gatewayData, ambientData, nodeData, tdrData, year, activeCharts }) => {
   const [axisMinMaxGateway, setAxisMinMaxGateway] = useState([{ min: now }, { max: now }]);
   const [axisMinMaxTdr, setAxisMinMaxTdr] = useState([{ min: now }, { max: now }]);
-  let {
-    gatewayData,
-    activeCharts,
-    // nodeData,
-    tdrData,
-    year,
-  } = props;
+  const [axisMinMaxLitterbag, setAxisMinMaxLitterbag] = useState([{ min: now }, { max: now }]);
+  const [axisMinMaxNode, setAxisMinMaxNode] = useState([{ min: now }, { max: now }]);
 
   useEffect(() => {
-    const timestampsGateway = gatewayData.map((data) => new Date(data.timestamp).getTime());
-    setAxisMinMaxGateway({
-      min: convertEpochtoDatetime(new Date(Math.min(...timestampsGateway))),
-      max:
-        new Date().getFullYear().toString() === year
-          ? now
-          : convertEpochtoDatetime(new Date(Math.max(...timestampsGateway))),
-    });
+    const generateMinMax = (sensorData) => {
+      return {
+        min: sensorData.length === 0 ? now : convertEpochtoDatetime(sensorData[0].timestamp),
```

Comment:
this is where i just take the first timestamp because i already sorted it by timestamp